exercises/practice/all-your-base/.approaches/from-base-ten/content.md

```diff
@@ -0,0 +1,118 @@
+# Converting from base 10
```

Comment:
This is a pet peeve of mine: I've mentored this solution so many times and many students seem to think that an `int` is inherently "decimal". When they convert the input to a intermediate numeric value they refer to it as "decimal" and sometimes they even add unnecessary conversion steps because of that.

I would strongly prefer calling it a "numeric value" instead of "base 10", here an in the rest of the approaches.

Reply:
A numeral that is of base 10, the meaning of decimal.  While "numeric" is anything that is numeric, sometimes decimal, sometimes of a different base.

Since this is an exercise of bases, we should be triply clear on our wording concerning numbers.